fala/apps/adviser/tests/test_other_regions_playwright.py

```diff
@@ -10,6 +10,8 @@ def test_jersey(self):
         # We don't preserve Jersey postcodes search term, by design, as we don't want residents from Jersey to use our service
         expect(search_page.item_from_text("Organisation name")).to_be_visible()
 
+        self.test_failed_take_screenshot = False
```

Comment:
Okay there is a lot of repeat. I think it's worth looking into AfterEach with playwright. AfterEach should be called after each test is run. It can perform tasks like clean ups. in our case, we can check if the test failed or not to then take a screenshot.

Reply:
`AfterEach` from what I could tell was a node.js specific thing and did not help with out python set-up.